subsys/spm/spm.c

```diff
@@ -374,7 +382,7 @@ void spm_jump(void)
 	/* Extract initial MSP of the Non-Secure firmware image.
 	 * The assumption is that the MSP is located at VTOR_NS[0].
 	 */
-	u32_t *vtor_ns = (u32_t *)DT_FLASH_AREA_IMAGE_0_NONSECURE_OFFSET_0;
+	u32_t *vtor_ns = (u32_t *)PM_APP_ADDRESS;
```

Comment:
Where is this one define, could you please, indicate?

Reply:
It is defined in the generated file pm_config.h.

pm_config.h is generated by zephyr/scripts/partition_manager.py.

What kind of comment do you think would clarify this?

Something like this perhaps?

```
/* Extract initial MSP of the Non-Secure firmware image.  * The
* assumption is that the MSP is located at VTOR_NS[0], which comes
* from pm_config.h.
*/

```

Reply:
What happens if you want to run SPM without partition manager? is this possible?

Reply:
If SPM is built as a standalone app, partition manager will still handle the placement and size due to the existence of the `pm.yml` file.

Since no other images are present, the address will always be 0 though, and the size will be what is specified in Kconfig.

Reply:
I added the comment @SebastianBoe  proposed.

Reply:
Then it looks to me this rules-out the possibility of building images separately.

Reply:
I have updated the PR now so that partition manager is disabled completely when building a sub-image by itself. For instance by building the spm sample individually.

Reply:
Note that I reverted the comment update that @SebastianBoe proposed, as now the address if fetched from either DT or PM.